internal/antewrapper/msg_blacklist.go

```diff
@@ -0,0 +1,40 @@
+package antewrapper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+)
+
+// MsgTypeBlacklistContextDecorator is (hopefully) a temporary hard-coded antehandler that disallows certain messages.
+// Once the circuit breaker module is added to provenance, this should be removed.
+type MsgTypeBlacklistContextDecorator struct {
+	Blacklist []string
+}
+
+func NewMsgTypeBlacklistContextDecorator() MsgTypeBlacklistContextDecorator {
+	return MsgTypeBlacklistContextDecorator{
+		Blacklist: []string{
+			// Disallow vesting account creation due to barberry: https://forum.cosmos.network/t/cosmos-sdk-security-advisory-barberry/10825
+			// Once that fix is in the SDK that we pull in, these can be removed.
```

Comment:
We should make issues for comments like these in the future and maybe put the issue link in the comment.

Reply:
If this were going to `main`, I would make such an issue. But since it'll (hopefully) only be in `v1.15.2`, we won't actually have to remove it. But I put the comment there in the off chance it somehow makes its way back to `main`.

Reply:
Ok, makes sense.  Thought we would merge the deny list ante handler since it seems good to have around.

Reply:
we should keep this out of main in general because the circuit breaker stuff is coming and it does the same thing but better.